tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/NativeMobile/NativeMobileFirst/Supplier Get NativeMobileFirst.feature

```diff
@@ -24,7 +24,7 @@ Scenario: Native Mobile First is retreived for the solution
     And the string value of element mobile-first-design is Yes
 
 @3611
-Scenario: Native Mobile First is retrieved for the solution where no solutiondetail exists
+Scenario: Native Mobile First design is retrieved empty for the solution
```

Comment:
Suggestion: `native mobile first design is not retrieved when no mobile first design details exist`.

Reply:
Updated scenario description to `native mobile first design is not retrieved when no mobile first design details exist`